src/actions/definitionActions.js

```diff
@@ -29,17 +31,37 @@ export function getDefinitionAction(token, entity, name) {
   }
 }
 
-export function getDefinitionsAction(token, entities) {
+export function getDefinitionsAction(token, entities, isMissedDefinition = false) {
   return dispatch => {
     const actions = asyncActions(DEFINITION_BODIES)
     dispatch(actions.start())
     return getDefinitions(token, entities).then(
-      result => dispatch(actions.success({ add: result })),
+      result => {
+        if (isMissedDefinition) {
+          uiInfo(dispatch, 'Some definitions have been harvested. You are now able to check them out.')
+        }
+        dispatch(actions.success({ add: result }))
+      },
       error => dispatch(actions.error(error))
     )
   }
 }
 
+export function checkForMissingDefinition(token, isFirstAttempt = false) {
+  return (dispatch, getState) => {
+    const missingDefinitions = map(
+      get(getState(), 'definition.bodies.entries'),
+      (item, key) => !get(item, 'described.tools') && key
+    ).filter(x => x)
+    if (missingDefinitions.length > 0) {
+      if (isFirstAttempt)
+        uiInfo(dispatch, "Some definitions are not already harvested, we'll notice you once the operation will finish")
```

Comment:
Let's change it from
`"Some definitions are not already harvested, we'll notice you once the operation will finish"`
to
`'Some definitions are not already harvested, we'll notice you once the operation finishes.'`